src/interactivity/base/InteractivityFactory.cpp

```diff
@@ -334,18 +416,18 @@ using namespace Microsoft::Console::Interactivity;
                 const auto exStyles = WS_EX_TOOLWINDOW | WS_EX_TRANSPARENT | WS_EX_LAYERED | WS_EX_NOACTIVATE;
 
                 // Attempt to create window.
-                hwnd = CreateWindowExW(exStyles,
-                                       reinterpret_cast<LPCWSTR>(windowClassAtom),
-                                       nullptr,
-                                       windowStyle,
-                                       0,
-                                       0,
-                                       0,
-                                       0,
-                                       owner,
-                                       nullptr,
-                                       nullptr,
-                                       this);
+                _pseudoConsoleWindowHwnd = hwnd = CreateWindowExW(exStyles,
+                                                                  reinterpret_cast<LPCWSTR>(windowClassAtom),
+                                                                  nullptr,
+                                                                  windowStyle,
+                                                                  0,
+                                                                  0,
+                                                                  0,
+                                                                  0,
+                                                                  owner,
+                                                                  nullptr,
+                                                                  nullptr,
+                                                                  this);

```

Comment:
you can reduce the diffs here by assigning `_pseudoConsoleHwnd` to the value of `hwnd` later on line R437 :)